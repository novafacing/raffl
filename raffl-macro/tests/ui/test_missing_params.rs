use raffl_macro::callback_wrappers;

struct Test {}

#[callback_wrappers]
impl Test {
    pub fn test(&self, a: i32, b: i32) -> i32 {
        a + b
    }
}

fn main() {}
