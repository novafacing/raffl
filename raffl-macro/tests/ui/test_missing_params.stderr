error: expected `#[params(...)]`
 --> tests/ui/test_missing_params.rs:7:5
  |
7 | /     pub fn test(&self, a: i32, b: i32) -> i32 {
8 | |         a + b
9 | |     }
  | |_____^
